lms/static/scripts/frontend_apps/components/EmailNotificationsPreferences.tsx

```diff
@@ -15,18 +15,51 @@ const dayNames: [WeekDay, string][] = [
   ['sat', 'Saturday'],
 ];
 
+export type SaveOptions = {
+  selectedDays: EmailNotificationsPreferences;
+  event: Event;
```

Comment:
Yes, but for some reason `onSubmit` defines the handler with `Event`, not `SubmitEvent` 😅 

Perhaps we could do an inline casting so that the props API is more clear.